guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc

```diff
@@ -25,3 +25,18 @@ Use the `{foreman-installer}` command to configure {Project} to connect to an ex
   --foreman-proxy-content-pulpcore-postgresql-db-name pulpcore \
   --foreman-proxy-content-pulpcore-postgresql-password _Pulpcore_Password_
 ----
++
+
+To enable the Secure Sockets Layer (SSL) protocol for these external databases, add the following options:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+--foreman-db-sslmode verify-full
+--foreman-db-root-cert <path_to_CA>
+--katello-candlepin-db-ssl true
+--katello-candlepin-db-ssl-verify true
```

Comment:
This is the default value

Reply:
Hi @ekohl  , thank you for your review! Apologies, I'm not sure whether your comments just state the facts or are meant as requirements for further changes in the file I've edited. Please, are there any additional changes needed so that this PR can be merged? Just FYI, exactly the same changes have been confirmed as correct and already merged to branch 2.5 (Satellite 6.10).  Thank you!

Reply:
They go together with:
> I'm not sure about specifying the default values again. While they don't hurt, as an installer maintainer I'd feel safe leaving them out.

This is marking which values I'm talking about.